posthog/models/alert.py

```diff
@@ -96,18 +88,33 @@ class AlertConfiguration(CreatedMetaFields, UUIDModel):
         related_name="alert_configurations",
     )
 
+    # The series from the insight to evaluate the alert against.
+    series_index = models.IntegerField()
```

Comment:
Should it be blank=True and null=True? When you add other insights, you'll need to update it anyway.

Also, series_index seems to be related to Trends only. I'm wondering if we need a flexible configuration, but it might not be necessary because we probably need only Trends and Funnels in Alerts. What do you think?

Reply:
Haha, I pushed your recommended change just while you were commenting. Agreed, created a `config` field which for now has Trends specific fields.